stdlib/Sockets/src/Sockets.jl

```diff
@@ -709,16 +709,16 @@ end
 const localhost = ip"127.0.0.1"
 
 """
-    listenany([host::IPAddr,] port_hint) -> (UInt16, TCPServer)
+    listenany([host::IPAddr,] port_hint; backlog::Integer=BACKLOG_DEFAULT) -> (UInt16, TCPServer)
 
 Create a `TCPServer` on any port, using hint as a starting point. Returns a tuple of the
 actual port that the server was created on and the server itself.
 """
```

Comment:
Perhaps we could add a description of what the new parameter `backlog` does.

*libuv*
> backlog indicates the number of connections the kernel might queue, same as [listen(2)](https://man7.org/linux/man-pages/man2/listen.2.html).
> [uv_stream_t — Stream handle — libuv documentation](http://docs.libuv.org/en/v1.x/stream.html#c.uv_listen)

Or *listen(2)*
> The backlog argument defines the maximum length to which the queue of pending connections for sockfd may grow.
> [listen(2) - Linux manual page](https://man7.org/linux/man-pages/man2/listen.2.html#DESCRIPTION)

Or just `See [listen](@ref)`

Reply:
Added.